pkg/ipamd/ipamd.go

```diff
@@ -1355,7 +1357,8 @@ func (c *IPAMContext) verifyAndAddPrefixesToDatastore(eni string, attachedENIIPs
 		}
 
 		// Try to add the IP
-		err := c.dataStore.AddIPv4PrefixToStore(eni, strPrivateIPv4)
+		_, ipv4CidrPtr, _ := net.ParseCIDR(strPrivateIPv4)
```

Comment:
Probably want to check for err return here, otherwise we're going to get a crash when we dereference a nil `ipv4CidrPtr` on the next line.